src/main/kotlin/no/nav/medlemskap/services/pdl/HentPdlPersonResponse.kt

```diff
@@ -34,6 +36,19 @@ enum class KjoennType{
     UKJENT,
 }
 
+data class Familierelasjon(
+        val relatertPersonIdent: String,
```

Comment:
Superpirk: relatertPersonsIdent (samme som PDL)

Reply:
Dette viste seg å ikke være superpirk, for det var feil i PDL-klassen også..